### PR TITLE
OSSM-3412: Service Mesh 2.3.2 z stream release notes

### PR DESCRIPTION
**Pull request by gwynnemonahan**
Version(s):
4.9+

Issue:
https://issues.redhat.com/browse/OSSM-3412

Link to docs preview:
New Features: https://56501--docspreview.netlify.app/openshift-enterprise/latest/service_mesh/v2x/servicemesh-release-notes.html#new-features-red-hat-openshift-service-mesh-version-2-3-2

Fixed Issues: https://56501--docspreview.netlify.app/openshift-enterprise/latest/service_mesh/v2x/servicemesh-release-notes.html#ossm-rn-fixed-issues-ossm_ossm-release-notes

Specific Fixed Issues to review for this release:

[OSSM-2493](https://issues.redhat.com//browse/OSSM-2493)
[OSSM-2492](https://issues.redhat.com//browse/OSSM-2492)
[OSSM-2374](https://issues.redhat.com//browse/OSSM-2374)
[OSSM-2373](https://issues.redhat.com//browse/OSSM-2373)
[OSSM-2371](https://issues.redhat.com//browse/OSSM-2371)

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by fjglira** [comment]
From QE LGTM

**Comment by leandroberetta** [comment]
Hi, @gwynnemonahan this looks good to me too.

File: modules/ossm-rn-fixed-issues.adoc [hunk 1]
```diff
@@ -21,6 +21,18 @@ The following issues been resolved in the current release:
 
 * https://issues.redhat.com/browse/OSSM-3025[OSSM-3025] Istiod sometimes fails to become ready. Sometimes, when a mesh contained many member namespaces, the Istiod pod did not become ready due to a deadlock within Istiod. The deadlock is now resolved and the pod now starts as expected.
 
+* https://issues.redhat.com/browse/OSSM-2493[OSSM-2493] Default nodeSelector and tolerations in SMCP not passed to Kiali. The nodeSelector and tolerations you add to `SMCP.spec.runtime.defaults` are now passed to the Kiali resource.
+
+* https://issues.redhat.com/browse/OSSM-2492[OSSM-2492] Default tolerations in SMCP not passed to Jaeger. The nodeSelector and tolerations you add to `SMCP.spec.runtime.defaults` are now passed to the Jaeger resource.
```
Comment (luksa) [inline modules/ossm-rn-fixed-issues.adoc:26]:
```suggestion
* https://issues.redhat.com/browse/OSSM-2492[OSSM-2492] Default tolerations in SMCP not passed to Jaeger. The `nodeSelector` and `tolerations` you add to `SMCP.spec.runtime.defaults` are now passed to the Jaeger resource.
```

File: modules/ossm-rn-fixed-issues.adoc [hunk 2]
```diff
@@ -21,6 +21,18 @@ The following issues been resolved in the current release:
...
+
+* https://issues.redhat.com/browse/OSSM-2374[OSSM-2374] Deleting a SMM also deletes the SMMR in OpenShift Service Mesh. The Service Mesh operator no longer deletes the `ServiceMeshMemberRoll` resource when a `ServiceMeshMember` resource is removed manually.
```
Comment (luksa) [inline modules/ossm-rn-fixed-issues.adoc:28]: This needs to be reworded. This bugfix is about an issue where the operator deleted the ServiceMeshMemberRoll when the user deleted one of the ServiceMeshMembers associated with it. While this is the expected behavior when you delete the last ServiceMeshMember, the operator erroneously deleted the ServiceMeshMemberRoll even when the deleted ServiceMeshMember wasn't the last. This is the issue that is now fixed.

File: modules/ossm-rn-new-features.adoc [hunk 3]
```diff
@@ -15,11 +15,33 @@ Module included in the following assemblies:
 
 This release adds improvements related to the following components and concepts.
 
+== New features {SMProductName} version 2.3.2
+
+This release of {SMProductName} introduces new features, addresses Common Vulnerabilities and Exposures (CVEs), contains bug fixes, and is supported on {product-title} 4.9 or later.
```
Comment (luksa) [inline modules/ossm-rn-new-features.adoc:20]: I don't think we introduced any new features. It's only bug and CVE fixes. But I'd like others to confirm.

**Review by luksa — COMMENTED** [review]
A few minor comments.

**Review by luksa — APPROVED** [review]
lgtm

File: modules/ossm-rn-fixed-issues.adoc [hunk 4]
```diff
@@ -21,6 +21,18 @@ The following issues been resolved in the current release:
 
 * https://issues.redhat.com/browse/OSSM-3025[OSSM-3025] Istiod sometimes fails to become ready. Sometimes, when a mesh contained many member namespaces, the Istiod pod did not become ready due to a deadlock within Istiod. The deadlock is now resolved and the pod now starts as expected.
```
Comment (rh-tokeefe) [inline modules/ossm-rn-fixed-issues.adoc:23]: You should add the backticks to nodeSelector and tolerations

* https://issues.redhat.com/browse/OSSM-2493[OSSM-2493] Default `nodeSelector` and `tolerations` in SMCP not passed to Kiali. The nodeSelector and tolerations you add to `SMCP.spec.runtime.defaults` are now passed to the Kiali resource.

File: _attributes/common-attributes.adoc [hunk 5]
```diff
@@ -133,8 +133,8 @@ endif::[]
 :product-dedicated: Red Hat OpenShift Dedicated
 :SMProductName: Red Hat OpenShift Service Mesh
```
Comment (rh-tokeefe) [inline _attributes/common-attributes.adoc:134]: While the MaistraVersion and the SMProductVersion are both content variables, they are used in different places and serve slightly different purposes. The MaistraVersion should remain 2.3 or some code examples won't work.

I know this is confusing as you're learning :)

**Pull request merged**